### PR TITLE
Patch 6

### PR DESCRIPTION
**Pull request by Jasdemi**

### CODE REVIEW

File: ETC_en.tsv [hunk 1]
```diff
@@ -5128,31 +5128,31 @@ ETC_20150317_005127	My work shift ends soon!
 ETC_20150317_005128	Please defeat Stone Froster and obtain Nuclear Frost!
 ETC_20150317_005129	What?
 ETC_20150317_005130	What is that sound?
-ETC_20150317_005131	Please defeat Gargoyle!
-ETC_20150317_005132	Please defeat Devilglove!
```
Comment (ceddc) [inline ETC_en.tsv:5132]: Devilglove seems to be a boss name : 
ITEM_20150317_001983    Devilglove Card

File: ETC_en.tsv [hunk 2]
```diff
@@ -5128,31 +5128,31 @@ ETC_20150317_005127	My work shift ends soon!
...
 ETC_20150317_005140	Magic regulating valve
 ETC_20150317_005141	Scattered books
 ETC_20150317_005142	Scattered books
 ETC_20150317_005143	Hahahahaha....
-ETC_20150317_005144	Chase Grita who took the Essence of Fire!
+ETC_20150317_005144	Chase the Grita who took the Essence of Fire!
 ETC_20150317_005145	Well then.
 ETC_20150317_005146	Magic Circle
-ETC_20150317_005147	Helga Cercle
+ETC_20150317_005147	Helga Circle
```
Comment (ceddc) [inline ETC_en.tsv]: Helga cercle seems to be a boss name, but spelled differently in files : 
ITEM_20150317_002047    Helgacercle Card
asked question here :https://github.com/Treeofsavior/EnglishTranslation/issues/308

File: ETC_en.tsv [hunk 3]
```diff
@@ -5164,24 +5164,24 @@ ETC_20150317_005163	Please defeat Moyabruka!
...
 ETC_20150317_005166	So you are still alive...
-ETC_20150317_005167	Your role is done.{nl}Die in silence!
+ETC_20150317_005167	You're done.{nl}Die in silence!
 ETC_20150317_005168	Goddess Shaule
 ETC_20150317_005169	Restraining Magic Circle
 ETC_20150317_005170	Temple Border
 ETC_20150317_005171	Goddess Shaule
 ETC_20150317_005172	Portal
-ETC_20150317_005173	Please defeat Golem!
+ETC_20150317_005173	Please defeat the Golem!
```
Comment (ceddc) [inline ETC_en.tsv]: Golem seems to be a boss name : 
ITEM_20150317_001976    Golem Card

File: ETC_en.tsv [hunk 4]
```diff
@@ -5164,24 +5164,24 @@ ETC_20150317_005163	Please defeat Moyabruka!
...
 ETC_20150317_005174	Please defeat Spanashon!
 ETC_20150317_005175	Please defeat Yekub!
 ETC_20150317_005176	Patient
 ETC_20150317_005177	Trigger to kill soldiers
-ETC_20150317_005178	Please defeat Yonazolem and obtain the easy burning stone!
```
Comment (ceddc) [inline ETC_en.tsv:5178]: There is "please defeat" xxx everywhere . Seems to be an objective.
I guess it must be "Defeat" or "Please defeat" everywhere

File: ETC_en.tsv [hunk 5]
```diff
@@ -5164,24 +5164,24 @@ ETC_20150317_005163	Please defeat Moyabruka!
...
+ETC_20150317_005178	Defeat Yonazolem and obtain the faintly burning stone!
 ETC_20150317_005179	Let the fight begin!
 ETC_20150317_005180	Mines Manager
-ETC_20150317_005181	See? No just a day or two...
+ETC_20150317_005181	See? Not just a day or two...
 ETC_20150317_005182	Argh! There it is!!!
-ETC_20150317_005183	Please defeat Shadowgaler
-ETC_20150317_005184	Please defeat Minotaur!
+ETC_20150317_005183	Please defeat the Shadowgaler
+ETC_20150317_005184	Please defeat the Minotaur!
```
Comment (ceddc) [inline ETC_en.tsv]: Seems to be boss names : 
ITEM_20150317_002003    Minotaur Card
ITEM_20150317_002015    Shadowgaler Card

File: ETC_en.tsv [hunk 6]
```diff
@@ -5200,24 +5200,24 @@ ETC_20150317_005199	Don't come closer!
...
 ETC_20150317_005202	I will not go easy on you!
-ETC_20150317_005203	Please defeat Rogue Master!
+ETC_20150317_005203	Please defeat the Rogue Master!
 ETC_20150317_005204	Here I go!
 ETC_20150317_005205	Please defeat Rocktortuga and obtain the natural agate stone!
-ETC_20150317_005206	Please defeat Schwarzer Reiter!
+ETC_20150317_005206	Please defeat the Schwarzer Reiter!
 ETC_20150317_005207	Please defeat Deadbone!
-ETC_20150317_005208	Please defeat Rajatoad!
+ETC_20150317_005208	Please defeat the Rajatoad!
```
Comment (ceddc) [inline ETC_en.tsv]: There is a Rajatoad card (so boss name), but no Rogue Master or Schwarzer Reiter

**Comment by Jasdemi** [comment]
Thanks, ceddc. Changes made. I agree about the "Defeat" or "Please defeat". Just "Defeat" sounds better tbh. Feel free to change it after the pull :D

File: ETC_en.tsv [hunk 7]
```diff
@@ -5590,13 +5590,13 @@ ETC_20150317_005589	Town Street
...
 ETC_20150317_005592	See Water Forest
-ETC_20150317_005593	Curernt Fire Place
+ETC_20150317_005593	Current Fire Place
 ETC_20150317_005594	Large Flower Garden
 ETC_20150317_005595	Small Flower Garden
 ETC_20150317_005596	Heojeopyeong
 ETC_20150317_005597	Hyeonga Farm
 ETC_20150317_005598	Bare hands
-ETC_20150317_005599	Epee garde
+ETC_20150317_005599	Epee Garden
```
Comment (ceddc) [inline ETC_en.tsv]: Epee garde seems to be the name of a skill : 에페 갸르드 , seems garde is taken from French fencing.
There is the same skill in granado espada 
http://ge.jupath.me/GranadoEspadaUSA-Jobs-Fighter.html#EpeeGarde

File: ETC_en.tsv [hunk 8]
```diff
@@ -3854,35 +3854,35 @@ ETC_20150317_003853	ClassName is missing.
...
-ETC_20150317_003864	Unable to acquire quest item because inventory is full! {nl}
+ETC_20150317_003864	Unable to acquire quest item because your inventory is full!
 ETC_20150317_003865	Acquired quest item
 ETC_20150317_003866	Quest enrolled in the Session object QuestMapPointViewTerms
 ETC_20150317_003867	Quest enrolled in the Session object QuestMonViewTerms
 ETC_20150317_003868	Already used
-ETC_20150317_003869	Blessing of the Goddess {nl} Increased Skill Point by 1
-ETC_20150317_003870	You have already received Blessing of the Goddess
-ETC_20150317_003871	Trying to move to warp area that does not exist
+ETC_20150317_003869	Blessed by the Goddess {nl} Increased Skill Point by 1
```
Comment (ceddc) [inline ETC_en.tsv]: isn't it some buff or "item" description?
like "Blessing of the Goddess"

File: ETC_en.tsv [hunk 9]
```diff
@@ -3767,7 +3767,7 @@ ETC_20150317_003766	Thick clothes
 ETC_20150317_003767	wearing at least {/} armor will be of help
 ETC_20150317_003768	Thick Overclothes
 ETC_20150317_003769	wearing at least {/} armor will be of help
-ETC_20150317_003770	Use key to open inventory and wear using right mouse click
+ETC_20150317_003770	Open your inventory and right click an item to equip
```
Comment (Sunaries) [inline ETC_en.tsv]: right click -> right-click

File: ETC_en.tsv [hunk 10]
```diff
@@ -4160,34 +4160,34 @@ ETC_20150317_004159	Do you want to equip it to ]?
...
+ETC_20150317_004166	Female characters cannot equip this item. Check the gender and level requirements in the item description
+ETC_20150317_004167	Male characters cannot equip this item. Check the gender and level requirements in the item description
 ETC_20150317_004168	Request Contribution :
 ETC_20150317_004169	Please enter the changed value.
 ETC_20150317_004170	The ClassName does already exist
 ETC_20150317_004171	{ol}{@st41b} Silver that will receive : {Auto_1} {img Zeny 20 20}
 ETC_20150317_004172	{Img {Auto_1} 20 20} {ol} {@st45w3} {s18} {Auto_2} Buff
 ETC_20150317_004173	Function is not nagging class :
-ETC_20150317_004174	Belongings is full. {nl}
+ETC_20150317_004174	Your Inventory is full. {nl}
```
Comment (Sunaries) [inline ETC_en.tsv]: Inventory should be uncapitalized.

File: ETC_en.tsv [hunk 11]
```diff
@@ -4160,34 +4160,34 @@ ETC_20150317_004159	Do you want to equip it to ]?
...
 ETC_20150317_004181	{nl} Shortcut: {#0000FF}
-ETC_20150317_004182	You can use the potion items, you will be able to recover the {nl} SP.
+ETC_20150317_004182	You can use the potion items, you will be able to recover the SP.
 ETC_20150317_004183	You can use the potion items you will be able to recover stamina.
-ETC_20150317_004184	{s18}{#001100}You can raise your character's stats {nl} when leveling up.
+ETC_20150317_004184	{s18}{#001100}You can raise your character's stats {nl} by leveling up.
 ETC_20150317_004185	Click apply {nl} after increasing stats by using buttons.
-ETC_20150317_004186	{s18}{#001100}{a @UI_TUTO_MORU} Please enhance the items to right click on the anvil
-ETC_20150317_004187	{s18}{#001100} Instant stores the price more expensive {nl} than the regular stores.
+ETC_20150317_004186	{s18}{#001100}{a @UI_TUTO_MORU} Right click the anvil to enhance items
```
Comment (Sunaries) [inline ETC_en.tsv]: Right click -> Right-click

File: ETC_en.tsv [hunk 12]
```diff
@@ -4234,7 +4234,7 @@ ETC_20150317_004233	{@st41} Gesture List
 ETC_20150317_004234	[Main]
 ETC_20150317_004235	{@st55_b}{b} Reward(s) {/}
 ETC_20150317_004236	{@st55_c} Experience :
-ETC_20150317_004237	Waiting for the other to accept.
+ETC_20150317_004237	Waiting for the the player to accept.
```
Comment (Sunaries) [inline ETC_en.tsv]: Replace the second 'the' with 'other'. 
So: Waiting for the other player to accept.

File: ETC_en.tsv [hunk 13]
```diff
@@ -4495,22 +4495,22 @@ ETC_20150317_004494	Hope
 ETC_20150317_004495	Avarice
 ETC_20150317_004496	Fit.
 ETC_20150317_004497	Was wrong.
-ETC_20150317_004498	Call the brave Gedulah of jakariel by reactivating of altar of gedulah.
+ETC_20150317_004498	Reactivate the Altar of Gedulah to call the brave Gedulah from Jakariel.
```
Comment (Sunaries) [inline ETC_en.tsv]: Jakariel -> Zachariel

File: ETC_en.tsv [hunk 14]
```diff
@@ -4495,22 +4495,22 @@ ETC_20150317_004494	Hope
...
 ETC_20150317_004499	Wisdom
 ETC_20150317_004500	Gold
 ETC_20150317_004501	Food
 ETC_20150317_004502	The Great King is waiting for you.
-ETC_20150317_004503	Call the faithful Sviesa of Jakariel by reactivating of altar of Sviesa
+ETC_20150317_004503	Reactivate the Altar for Sviesa to call the faithful Sviesa from Jakariel.
```
Comment (Sunaries) [inline ETC_en.tsv]: Jakariel -> Zachariel

File: ETC_en.tsv [hunk 15]
```diff
@@ -4798,8 +4798,8 @@ ETC_20150317_004797	Great eyes have whispered you to the right person is a Waton
 ETC_20150317_004798	Part of the fragmentation barrier is insufficient!
 ETC_20150317_004799	Junior Spirit
 ETC_20150317_004800	Works I was fully collected!
-ETC_20150317_004801	Saule goddess. I will need your protection.
-ETC_20150317_004802	Saule goddess. Please the power to overcome my crisis
+ETC_20150317_004801	Saule Goddess, please protect me
```
Comment (Sunaries) [inline ETC_en.tsv]: Saule Goddess -> Goddess Saule

File: ETC_en.tsv [hunk 16]
```diff
@@ -4285,20 +4285,20 @@ ETC_20150317_004284	{A TOGGLE_PCMACRO_COND}{ul} and {/}{/}
 ETC_20150317_004285	Please select the operation of one or more.
 ETC_20150317_004286	Please enter a name for the rule one or more characters.
 ETC_20150317_004287	[{Auto_1}]: This rule name is already exists.
-ETC_20150317_004288	{s22} Do you want to really give up the request?
+ETC_20150317_004288	{s22} Do you really want to abandon this quest?
```
Comment (Sunaries) [inline ETC_en.tsv]: The Korean string doesn't mention 'quest', but 'request'.

File: ETC_en.tsv [hunk 17]
```diff
@@ -4700,7 +4700,7 @@ ETC_20150317_004699	Shop not for sale
 ETC_20150317_004700	Individual tradable
 ETC_20150317_004701	Individual transaction impossible
 ETC_20150317_004702	And when attached to the card socket,
-ETC_20150317_004703	Anvil does not match the type of equipment
```
Comment (Sunaries) [inline ETC_en.tsv:4703]: "This anvil does not match this type of equipment"

File: ETC_en.tsv [hunk 18]
```diff
@@ -4850,32 +4850,32 @@ ETC_20150317_004849	Caught more than five!
...
+ETC_20150317_004864	Removed the summonning circle of the demon!
+ETC_20150317_004865	Hurry up! Activate the barrier!
 ETC_20150317_004866	Huff and Puff
 ETC_20150317_004867	Collected fresh water!
-ETC_20150317_004868	The request has been already accepted
-ETC_20150317_004869	Today, It is already running request
-ETC_20150317_004870	Accepted the requests
-ETC_20150317_004871	You have exceeded the request to be carry
-ETC_20150317_004872	You have exceeded your accept limit of daily request
-ETC_20150317_004873	Start Request
```
Comment (Sunaries) [inline ETC_en.tsv:4873]: You've changed these few strings where you change 'request' to 'quest'. The Korean strings don't mention quests. It actually does say request.

File: ETC_en.tsv [hunk 19]
```diff
@@ -5017,18 +5017,18 @@ ETC_20150317_005016	Is there anyone who can help?!
...
-ETC_20150317_005021	{@st41}Open your inventory by pressing the {img F2 40 40} key to use the EXP card. {nl} {@st50} The {img icon_item_expcard 40 40} EXP card will reward you with experience points. {/}
+ETC_20150317_005020	You obtained the records about the Ruklys era.
+ETC_20150317_005021	{@st41}Open your inventory by pressing the {img F2 40 40} key and use the EXP card. {nl} {@st50} The {img icon_item_expcard 40 40} EXP card will reward you with experience points. {/}
 ETC_20150317_005022	The tombstone lanterns are being attacked by the guardians!
 ETC_20150317_005023	Bearkaras' suppressed power has returned!
 ETC_20150317_005024	Panto_Rsword: Hot Panto Fighter : 50/Panto_Rspear : Hot Panto Spearman: 50/Panto_Rhand : Hot Panto : 50/Panto_Rstaff : Hot Panto Wizard : 51
 ETC_20150317_005025	Crafting the item
 ETC_20150317_005026	Ask about the Revelator
 ETC_20150317_005027	Ask about the current situation
-ETC_20150317_005028	Ask about the Day of Shinsoo that took place four years ago
```
Comment (Sunaries) [inline ETC_en.tsv:5028]: This is wrong. Shinsoo is not a location. Should be changed to "Divine Day".

File: ETC_en.tsv [hunk 20]
```diff
@@ -5067,10 +5067,10 @@ ETC_20150317_005066	Oak Pattern
 ETC_20150317_005067	Rough Pottery
 ETC_20150317_005068	Navy
 ETC_20150317_005069	Tu-Tutu...! Help me!!!
-ETC_20150317_005070	Please defeat Unknocker!
+ETC_20150317_005070	Please defeat the Unknocker!
```
Comment (Sunaries) [inline ETC_en.tsv]: "Please defeat Unknocker!" or "Defeat Unknocker!" (depending if this piece of line is shown in NPC dialogue or quest UI) 

Unknocker is a boss monster.

File: ETC_en.tsv [hunk 21]
```diff
@@ -5128,31 +5128,31 @@ ETC_20150317_005127	My work shift ends soon!
 ETC_20150317_005128	Please defeat Stone Froster and obtain Nuclear Frost!
 ETC_20150317_005129	What?
 ETC_20150317_005130	What is that sound?
-ETC_20150317_005131	Please defeat Gargoyle!
+ETC_20150317_005131	Please defeat the Gargoyle!
```
Comment (Sunaries) [inline ETC_en.tsv]: See previous note of the Unknocker edit about this.

File: ETC_en.tsv [hunk 22]
```diff
@@ -5503,13 +5503,13 @@ ETC_20150317_005502	Defeated 500 boss monsters.
 ETC_20150317_005503	Walking Dead
 ETC_20150317_005504	Achieved by resurrecting at least 100 times using NXP.
 ETC_20150317_005505	Crafting Beginner
-ETC_20150317_005506	Succeeded in crafting 10 times. Feels pretty good.
```
Comment (Sunaries) [inline ETC_en.tsv:5506]: Not sure if you thought this was a misspelling, but it works either way. The Korean strings say 'times' as in: crafted an item 10 times

File: ETC_en.tsv [hunk 23]
```diff
@@ -5128,31 +5128,31 @@ ETC_20150317_005127	My work shift ends soon!
...
-ETC_20150317_005138	Flam Fusion Machine
+ETC_20150317_005137	Damn it. It was a trap. This is a Restrainment Spell.
+ETC_20150317_005138	Flame Fusion Machine
 ETC_20150317_005139	Have a good time
 ETC_20150317_005140	Magic regulating valve
 ETC_20150317_005141	Scattered books
 ETC_20150317_005142	Scattered books
 ETC_20150317_005143	Hahahahaha....
-ETC_20150317_005144	Chase Grita who took the Essence of Fire!
+ETC_20150317_005144	Chase the Grita who took the Essence of Fire!
```
Comment (Sunaries) [inline ETC_en.tsv]: 'the' is redundant.

File: ETC_en.tsv [hunk 24]
```diff
@@ -5859,7 +5858,7 @@ ETC_20150317_005858	Transpose Buff
 ETC_20150317_005859	Convert monster's HP into attack
 ETC_20150317_005860	During throw
 ETC_20150317_005861	During throw
-ETC_20150317_005862	A state of continuous flame damage.
+ETC_20150317_005862	Receiving continuous damage
```
Comment (Sunaries) [inline ETC_en.tsv]: Why did you remove the word 'flame'? It's flame damage.

**Comment by Sunaries** [comment]
Good job, but you need to check the notes I left you. It is very useful to read the Korean strings. Leaving notes on my phone took so long, grrr.

File: ETC_en.tsv (hunk 22 above)
Comment (Jasdemi) [inline ETC_en.tsv:5506]: I don't know. "crafting an item 10 times" sounds almost like you have to craft the same item 10 times. "by crafting 10 items" is simple and straightforward for the player, imo.

Thanks a lot for the notes, I have changed them all except this one. Will try to cross check the original Korean text next time. :)

**Comment by imcgames** [comment]
Thank you all guys! :+1:
